docker-compose.yml

```diff
@@ -23,3 +24,4 @@ services:
       - 3306:3306
     healthcheck:
       test: "exit 0"
+    restart: unless-stopped
```

Comment:
@djsoults suggested having two docker-compose files, one that handles `restart: unless-stopped` and one that doesn't, we should research to see if the ` restart: unless-stopped` file could inherit from the other.